### ISSUE TITLE
No Status Source for Elune

### ISSUE THREAD

**Issue by JLDNetwork** [ycfreeman/wow-server-status-widget#4]
Currently the widget is displaying "NO STATUS SOURCE". I have gone over everything multiple times and finally came to the conclusion that it isn't returning correctly due to multiple sources.

My realm is Elune. There is also a realm named Sisters of Elune. Both sources are being returned and as far as I can tell, the widget isn't tracking the chance at multiple resource returns.

**Comment by ycfreeman** [comment]
ya seems it's not working, i'll look into that later

**Comment by ycfreeman** [comment]
seems it's because blizzard has closed public realm status api. a change will be needed to take in an api key, i'll try to get the update in when i have time these few weeks

**Comment by JLDNetwork** [comment]
I have already been working on a fix using the battle.net api. I also have an api key to use the api. I have made a few successful data pulls using the api, so far so good. I have NOT implemented it into this plugin. Maybe the code I have will help resolve the issue.

```
 //-- Your Registered API Key --//
 $wowss['apikey'] = 'YOUR_KEY_GOES_HERE';

 //-- Fetch the data feed from Battle.net --//
 $json_wow_api_url = file_get_contents('https://'.$wowss['region'].'.api.battle.net/wow/realm/status?locale='.$wowss['locale'].'&apikey='.$wowss['apikey'].'&realms='.$wowss['realm'].'');  

 $obj = json_decode($json_wow_api_url);
 $realm_status['type'] = $obj->realms[0]->type;
 $realm_status['population'] = $obj->realms[0]->population;
 $realm_status['status'] = $obj->realms[0]->status;
 $realm_status['realm'] = $wowss['realm'];
```

The url is setup to only return a single realm. You can get more realms by adding them to the url, separated by a comma. You can return all realm data by removing the '&realms' from the url.

This method has proven to work consistently and apparently is the standard use as per Battle.net. Anyways, I hope this helps.

As for the variables used in this code snippet, they were all, obviously, being pulled from your plugin, for testing. 

Again, what I have successfully pulled the data from the battle.net api. I have not implemented the change (not real clear on what to change), so the plugin is still not working.

![2016-04-03](https://cloud.githubusercontent.com/assets/18234886/14233229/6cc542ae-f988-11e5-92dc-57ac191b49a6.png)

**Comment by ycfreeman** [comment]
finished the fix, releasing 1.0.13
